Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift

```diff
@@ -8,35 +8,60 @@
 
 import Foundation
 import SWXMLHash
+import RealmSwift
 
 class TurkishAirLinesHelper {
     
     enum GameDataStatus {
         case schedule
         case results
     }
-    //In viewModel
+    
+    //In viewModel has array of games according to target round
     var table: Dictionary<String, [Array<GameData>]> = [:]
     
-    private var games: Dictionary<Int, GameData> = [:]
+    private var games: Dictionary<Int, GameData> = [:] {
+        didSet{
+            //Notify the view Model
+        }
+    }
     
-    //In viewModel
+    //In viewModel has target round
     private var round: String = ""
     
     let urls: Dictionary<GameDataStatus, String> = [
         .schedule : "http://www.euroleague.net/euroleague/api/schedules?seasoncode=E2016",
         .results : "http://www.euroleague.net/euroleague/api/results?seasoncode=E2016"
     ]
     
+    //Should have no parameters and return table of dataBase
     func getGamesTable(round: String) {
-        self.round = round
-        if games.count == 0 {
-            getSchedule()
+        games.removeAll()
+        let table = RealmDBManager.sharedInstance.getDataFromRealm()
+        print("Data is here")
+        print("table count: \(table.count)")
+        for game in table {
+            let gameData = GameData()
```

Comment:
I think it is better to create copy method in GameData class that copy object to another one